x-pack/test/functional/config.js

```diff
@@ -164,7 +164,6 @@ export default async function ({ readConfigFile }) {
         `--server.uuid=${env.kibana.server.uuid}`,
         `--server.port=${servers.kibana.port}`,
         `--elasticsearch.url=${formatUrl(servers.elasticsearch)}`,
-        '--xpack.monitoring.kibana.collection.enabled=false',
```

Comment:
It wasn't my first choice to remove this, but it is necessary for making the api integration test work. The HTTP endpoint relies `collectorSet` being exposed, which is dependent on this setting being `true`.

Reply:
It's kind of unfortunate that all functional tests depend on the same configuration. Given the startup time of Kibana though, I get why that is the best option though.

Didn't we need to add this because it randomly broke other tests though?

Reply:
> Didn't we need to add this because it randomly broke other tests though?

Nope, this was just an optimization to avoid running the internal timer for the functional test server.